worlds/generic/docs/advanced_settings_en.md

```diff
@@ -225,8 +337,8 @@ triggers:
 * `start_location_hints` gives us a starting hint for the `Spike Cave` location available at the beginning of the
   multiworld that can be used for no cost.
 * `exclude_locations` forces a not important item to be placed on the `Cave 45` location.
-* `item_links` causes all players with the same `item_links` settings to share a `Fire Rod` and `Ice Rod`. Extra 
-  `Rupee (1)` are put in the item pool instead of additional Rods.
+* `item_links` causes all players with the same `item_links` settings to share a `Fire Rod` and `Ice Rod`. Extra
```

Comment:
The Link to the Past and Timespinner item_links are two separate sections and should be documented as such.

Reply:
this was not fixed correctly.

Reply:
I think I fixed it now. Let me know if there is any other way to improvements that you suggest.